.circleci/config.yml

```diff
@@ -54,6 +54,7 @@ jobs:
       - run:
           name: Test Setup
           command: |
+            yarn build
```

Comment:
Do we have feature specs in this repository? I'd wonder if we should be running `rake assets:precompile` (including Sass compilation) as well / instead.

Reply:
Yeah, we do. I can swap that one in